Demo/imdb/queries.py

```diff
@@ -0,0 +1,103 @@
+import sys
+
+sys.path.append('../../')
```

Comment:
Isn't ../../ too far back?
../ is Demo

Reply:
The Demo is in ../, but because the class I want to use is in the __init__.py file it doesn't recognize it. 
So I hacked it a bit, and done ../../ and the in the import done "from Demo import QueryInfo"